triton-vm/src/lib.rs

```diff
@@ -113,3 +113,66 @@ pub fn prove(
 pub fn verify(parameters: &StarkParameters, claim: &Claim, proof: &Proof) -> bool {
     Stark::verify(parameters, claim, proof, &mut None).unwrap_or(false)
 }
+
+#[cfg(test)]
+mod public_interface_tests {
+    use triton_opcodes::program;
+    use twenty_first::shared_math::bfield_codec::BFieldCodec;
+
+    use crate::stark::StarkHasher;
+
+    use super::*;
+
+    #[test]
+    pub fn lockscript_test() {
+        // Program proves the knowledge of a hash preimage
+        let source_code = "
+        divine divine divine divine divine
+        hash pop pop pop pop pop
+        push 09456474867485907852 push 12765666850723567758 push 08551752384389703074 push 03612858832443241113 push 12064501419749299924
+        assert_vector
+        read_io read_io read_io read_io read_io
+        halt";
+
+        let secret_input = vec![
+            7534225252725590272u64,
```

Comment:
```suggestion
            7534225252725590272,
```
The type can be inferred, even if the IDE might have trouble with it.